plugin/trino-delta-lake/src/test/resources/deltalake/stats_with_minmax_nulls/README.md

```diff
@@ -0,0 +1,13 @@
+Data generated using OSS Delta Lake 2.4.0 on Spark 3.4.0:
+
+```
+bin/spark-sql --packages io.delta:delta-core_2.12:2.4.0 \
+  --conf "spark.sql.extensions=io.delta.sql.DeltaSparkSessionExtension" \
+  --conf "spark.sql.catalog.spark_catalog=org.apache.spark.sql.delta.catalog.DeltaCatalog" \
+  --conf "spark.sql.jsonGenerator.ignoreNullFields=false"
+```
+
+```sql
+CREATE TABLE delta.`/tmp/stats-with-minmax-nulls` USING DELTA AS SELECT col1 as id, col2 as id2 FROM VALUES (0, 1),(1,2),(3, 4);
+INSERT INTO delta.`/tmp/stats-with-minmax-nulls` select null, null;
```

Comment:
optional: We could add another test reading a checkpoint parquet file having null value in min/max statistics.

Reply:
Added a test table with checkpoint. Currently, it just has stats as JSON string in the checkpoint. Delta OSS doesn't have the feature to write the parsed stats in checkpoint. Will followup with a test table with parsed stats in the checkpoint file in the next PR.